src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable {
+    protected String color;
+
+    public Figure(String color) {
```

Comment:
Creating a figure, don't pass expressions in the constructor. It's better to create a separate variable for 'color' and pass it on for better code readability.